src/test/resources/org/prebid/server/it/test-application.properties

```diff
@@ -152,6 +152,11 @@ adapters.triplelift.enabled=true
 adapters.triplelift.endpoint=http://localhost:8090/triplelift-exchange
 adapters.triplelift.pbs-enforces-gdpr=true
 adapters.triplelift.usersync.url=//triplelift-usersync
+adapters.tripleliftnative.enabled=true
+adapters.tripleliftnative.endpoint=http://localhost:8090/triplelift_native-exchange
+adapters.tripleliftnative.pbs-enforces-gdpr=true
+adapters.tripleliftnative.usersync.url=//triplelift-usersync
```

Comment:
Seems duplicates with `adapters.triplelift.usersync.url` value.
Not critical, but keep in mind.